galeracluster/source/migration.rst

```diff
@@ -51,3 +51,96 @@ For servers that use ``systemd``, instead use this command:
 Running this command stops database server.  When you are ready to initialize your cluster, choose this server as your starting node.  
 
 .. seealso:: For more information on initializing and adding nodes to a cluster, see :doc:`startingcluster`.
+
+---------------------------------------
+Migrating from MySQL to Galera Cluster
+---------------------------------------
+.. _`migrating-mysql-galera`:
+
+In the event that you have an existing database server using the MyISAM storage engine or the stock MySQL master-slave cluster, there are some additional steps that you must take in order to migrate your data to Galera Cluster.
+
+There are two stages to this migration: migrating the database state from the previous installation to Galera Cluster and migrating the MySQL installation on the former master node to Galera Cluster.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Data Migration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. _`data-migration`:
+
+The first stage of migration is to transfer the database state from the existing system to Galera Cluster.  Begin by creating a cluster.  For more information on how to do so, see :doc:`gettingstarted`.
+
+- For migration from a standalone MySQL server, create the cluster using only new nodes.
+
+- For migration from a stock MySQL master-slave cluster, create the cluster using only slave nodes.
+
+You now have Galera Cluster and a single MySQL server running together.  The MySQL server is hereafter referred to as the MyISAM master.  To migrate your data from the MyISAM master to Galera Cluster, complete the following steps:
+
+#. Stop all load on the MyISAM master.
+
+#. Run ``mysqldump`` to create a state snapshot.
+
+   .. code-block:: console
+
+      $ mysqldump --skip-create-options --all-databases > sst.sql
+
+   The ``--skip-create-options`` ensures that the newly created tables default to InnoDB.
+
+#. Transfer the ``sst.sql`` file to one of the Galera Cluster nodes, then load the data.
+
+   .. code-block:: mysql
+
+      LOAD DATA LOCAL INFILE '/path/to/sst.sql';
```

Comment:
LOAD DATA LOCAL INFILE can not be used to load SQL files. The mysql client is required to do that.